omnibus/package-scripts/agent/prerm

```diff
@@ -82,15 +82,33 @@ remove_py_compiled_files()
 remove_downloader_data()
 {
     # Remove all the data that was downloaded using the integration command
+    # This is purely for cleaning up after ourselves
     DOWNLOADER_DATA="$INSTALL_DIR/embedded/lib/python2.7/site-packages/datadog_checks/downloader/data"
     if [ -d $DOWNLOADER_DATA ]; then
         rm -rf $DOWNLOADER_DATA
     fi
 }
 
+uninstall_integrations()
+{
+    # Uninstall every datadog integration
+    PIP_PATH=$INSTALL_DIR/embedded/bin/pip
+    if [ -x $PIP_PATH ]; then
+        echo "Uninstalling integrations..."
+        $PIP_PATH freeze | grep ^datadog- | grep -v datadog-checks-base | xargs $PIP_PATH uninstall -y -q --no-cache-dir || true
+    fi
+}
+
 if [ -f "/etc/debian_version" ] || [ "$DISTRIBUTION" = "Debian" ] || [ "$DISTRIBUTION" = "Ubuntu" ]; then
     stop_agent
     deregister_agent
+    # Starting with 6.10, integrations are uninstalled on package removal
+    # Upgrade from 6.8 or 6.9 to 6.10 won't uninstall integrations because it was previously done in preinst
```

Comment:
Could you document this in the PR description instead of here? (since it's documenting a "transient" issue of upgrade from 6.8 or 6.9 to any 6.10+ version)